api/config/rolepermission.php

```diff
@@ -0,0 +1,521 @@
+        ],
+        'update-team-poolClosingDate' => [
+            'en' => 'Update Team Pool Closing Date',
+            'fr' => 'Mise à jour de la date de clôture de la réserve d\'équipes'
+        ],
+        'delete-team-draftPool' => [
+            'en' => 'Delete Team Draft Pool',
+            'fr' => 'Supprimer le pool de sélection d\'équipes'
+        ],
+
+        'view-own-application' => [
+            'en' => 'View Own Application',
+            'fr' => 'Afficher sa propre application'
+        ],
+        'view-team-submittedApplication' => [
+            'en' => 'View Team Submitted Application',
+            'fr' => 'Voir la demande soumise par l\'équipe'
+        ],
+        'view-team-applicantProfile' => [
+            'en' => 'View Team Applicant Profile',
+            'fr' => 'Voir le profil du candidat de l\'équipe'
+        ],
+        'create-own-draftApplication' => [
+            'en' => 'Create Own Draft Application',
+            'fr' => 'Créer son propre projet d\'application'
+        ],
+        'submit-own-application' => [
+            'en' => 'Submit Own Application',
+            'fr' => 'Soumettre sa propre demande'
+        ],
+        'update-team-applicationStatus' => [
+            'en' => 'Update Team Application Status',
+            'fr' => 'Mise à jour du statut de la demande de l\'équipe'
+        ],
+        'delete-own-draftApplication' => [
+            'en' => 'Delete Own Draft Application',
+            'fr' => 'Supprimer son propre projet de demande'
+        ],
+        'archive-own-submittedApplication' => [
+            'en' => 'Archive Own Submitted Application',
+            'fr' => 'Archiver sa propre demande'
+        ],
+        'view-any-applicantCount' => [
+            'en' => 'View Applicant Count',
+            'fr' => 'Voir le nombre de candidats'
+        ],
+
+        'create-any-searchRequest' => [
+            'en' => 'Create Any Search Request',
+            'fr' => 'Créer toute demande de recherche'
+        ],
+        'view-team-searchRequest' => [
+            'en' => 'Create Team Search Request',
+            'fr' => 'Créer une demande de recherche d\'équipe'
+        ],
+        'update-team-searchRequest' => [
+            'en' => 'Update Team Search Request',
+            'fr' => 'Mise à jour de la demande de recherche d\'équipe'
+        ],
+        'delete-team-searchRequest' => [
+            'en' => 'Delete Team Search Request',
+            'fr' => 'Supprimer une demande de recherche d\'équipe'
+        ],
+
+        'view-any-team' => [
+            'en' => 'View Any Team',
+            'fr' => 'Voir n\'importe quelle équipe'
+        ],
+        'view-any-teamUsersBasicInfo' => [
+            'en' => 'View Any Team Users Basic Info',
+            'fr' => 'Afficher les informations de base sur les utilisateurs de n\'importe quelle équipe'
+        ],
+        'view-team-teamUsersBasicInfo' => [
+            'en' => 'View Teams Team Users Basic Info',
```

Comment:
Will folks using this know what this means? Is there another way to write this?

Reply:
"View the BasicInfo of Users in my Teams" is my more descriptive attempt.

If you're interested, I could use some feedback on https://github.com/GCTC-NTGC/gc-digital-talent/issues/4362, which also deals with some how to break down (and name) different versions of `User`.